po/manifest2po

```diff
@@ -92,7 +92,7 @@ function process_menu(menu) {
         if (menu[m].label) {
             push({
                 msgid: menu[m].label,
-                locations: [ filename ]
+                locations: [ filename + ":0" ] // Fake line number to zero
```

Comment:
Well, that's the "what", which is quite obvious -- I meant document the "why", like "zanata crashes with BlahException when there is no line number"